src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Shape {
```

Comment:
better not to confuse with names
```suggestion
public enum Figure {
```